README.md

```diff
@@ -32,6 +32,10 @@ Or the secret names and AWS region:
 
   ```secrets.aws.k8s/region: <AWS region for the secrets>```
 
+Optional flag for exploding json format in secret manager into into individual file
```

Comment:
Add that this only works for exploding string type secrets (not binary) and that it applies to all secrets.